index.html

```diff
@@ -157,10 +157,28 @@ <h5 class="about-heading">About Me</h5>
                     </ul>
                 </li>
             </ul>
-            
         </section>
-             
     </section>
-
+    <footer >
+    <section class="form">
+        <h5 class="form-heading">I'm always<br>interested in hearing about new projects,
+         so if you'd like to chat please get in touch.</h5>
+        <form  class="form-inputs" action="https://formspree.io/f/xeqdvepo" method="post">
+        <input class="input" type="text" minlength="30" placeholder="Enter full name" required>
```

Comment:
- [ ] Kindly make sure your inputs are properly validated. I think there are names that only have 3 characters, so saying the `minlength=30` It is never works here..
`hint: You may consider removing that` :smile: